ert_shared/dark_storage/endpoints/compute/misfits.py

```diff
@@ -26,4 +32,33 @@ async def get_response_misfits(
     realization_index: Optional[int] = None,
     summary_misfits: bool = False,
 ) -> Response:
-    raise NotImplementedError
+
+    ensemble_name = get_name("ensemble", ensemble_id)
+    dataframe = data_for_key(ensemble_name, response_name)
+    if realization_index or realization_index == 0:
+        dataframe = pd.DataFrame(dataframe.loc[realization_index]).T
+
+    response_dict = {}
+    for index, data in dataframe.iterrows():
+        data_df = pd.DataFrame(data).T
+        response_dict[index] = data_df
+
+    obs = create_observations(res)
+    # TODO: is it ok to simply take the first element here?
```

Comment:
did we agree on something here? Throw some proper exception / assertion with a proper error msg?